include/xlnt/worksheet/sheet_view.hpp

```diff
@@ -274,6 +291,11 @@ class XLNT_API sheet_view
     /// The collection of selections
     /// </summary>
     std::vector<xlnt::selection> selections_;
+
+    /// <summary>
+    /// The zoom scale (percent), e.g. 100
+    /// </summary>
+    optional<int> zoom_scale_;
```

Comment:
There is no need to use an optional here. Just initialising zoom_sscale_ to 100 (its default value) is enough (similar to e.g. show_grid_lines_ is handled). This simplifies the logic.